src/Utils/arm/request.ts

```diff
@@ -100,7 +100,7 @@ export async function armRequest<T>({
 async function getOperationStatus(operationStatusUrl: string) {
   const response = await window.fetch(operationStatusUrl, {
     headers: {
-      Authorization: userContext.authorizationToken
+      Authorization: !userContext.authorizationToken ? "" : userContext.authorizationToken
```

Comment:
Same as other one

Reply:
I make the unit test and change in minor changes